libraries/chain/include/eosio/chain/apply_context.hpp

```diff
@@ -553,6 +554,8 @@ class apply_context {
       int  db_upperbound_i64( name code, name scope, name table, uint64_t id );
       int  db_end_i64( name code, name scope, name table );
 
+      db_context& db_get_context();
```

Comment:
General question, but is apply_context ever used in a const fashion?

Reply:
Technically, no.  It is used to call apply_context::db_*** methods and apply_context::idx***'s methods, which are all defined as non-const methods.  I'm putting a # warning to circle back when we have gotten past the hard stuff.